src/Model/BSONIterator.php

```diff
@@ -128,9 +128,7 @@ public function rewind()
         $this->advance();
     }
 
-    /**
-     * @see https://php.net/iterator.valid
-     */
+    /** @see https://php.net/iterator.valid */
```

Comment:
These fixes came from SlevomatCodingStandard.Commenting.RequireOneLineDocComment.

I don't mind them applying to `src/`. I think they make the most sense in `tests/` where we had a lot of `@dataProvider` annotations taking up multiple lines.